### ISSUE TITLE
Parameters passed to constructors are not counted in mapping stats

### ISSUE THREAD

**Issue by viktor40** [OrnitheMC/enigma#38]
The mapping stats in engigma can detect how many unmapped "things" there are. I counts classes, methods, fields and parameters separately. However, parameters passed to constructors are not counted in the parameters option.

**Comment by SpaceWalkerRS** [comment]
I will see if this affects upstream too, and if so, a bug fix should be submitted there

**Comment by SpaceWalkerRS** [comment]
it does appear to affect upstream too

**Comment by viktor40** [comment]
yep, I will look at fixing this bug and then we can submit it later.
